compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+
+  bootstrap:
+    build:
+      context: .
+      dockerfile: Dockerfile.libp2p
+    ports:
+      - "3000:3000"
+      - "8080:8080"
+    volumes:
+      - ./testnet:/etc/nomos
+    environment:
+      - NET_NODE_KEY=1000000000000000000000000000000000000000000000000000000000000000
+    entrypoint: sh
+    command: "/etc/nomos/run_bootstrap.sh"
+
+  libp2p-node:
+    build:
+      context: .
+      dockerfile: Dockerfile.libp2p
+    ports:
+      - "3001-4001:3000"
+      - "8081-9081:8080"
+    volumes:
+      - ./testnet:/etc/nomos
+    deploy:
+      replicas: 10 
+    depends_on:
+      - bootstrap
+      - etcd
+    environment:
+      - DOCKER_LIBP2P_REPLICAS=10
```

Comment:
You know you can paremetrize this with a variable you can use in `deploy.replicas` too?

Reply:
Yes, that would be optimal.